sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -857,6 +857,31 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
     assert(sql(s"list file $testFile").count() == 1)
   }
 
+  test("ADD ARCHIVE command") {
+    withTempDir { dir =>
+      val file = File.createTempFile("someprefix1", "somesuffix1", dir)
+      val zipFile = new File(dir, "test.zip")
+      TestUtils.createJar(Seq(file), zipFile)
+      sql(s"ADD ARCHIVE ${zipFile.getAbsolutePath}#foo")
+
+      val checkAddArchive =
+        sparkContext.parallelize(
+          Seq(
+            "foo",
+            s"foo/${file.getName}"), 1).map { name =>
+          (name, new File(SparkFiles.get(name)).canRead)
+        }.collect()
+
+      assert(checkAddArchive(0) === ("foo", true))
+      assert(checkAddArchive(1) === (s"foo/${file.getName}", true))
+      assert(sql("list archives").
```

Comment:
nit: please use the upper case where possible, `LIST ARCHIVES`.

Reply:
There are `list file`, `list files`, `list jar` and `list jars` in this file.
I don't think it's a matter but If you think it's really necessary them to be upper case, I'll fix including them.